src/Paket.Core/PackageMetaData.fs

```diff
@@ -93,10 +115,10 @@ let merge templateFile metaData =
     match templateFile with
     | { Contents = ProjectInfo(md, opt) } -> 
         let merged = 
-            { Id = md.Id ++ metaData.Id
-              Version = md.Version ++ metaData.Version
-              Authors = md.Authors ++ metaData.Authors
-              Description = md.Description ++ metaData.Description }
+            { Id = metaData.Id ++ md.Id
```

Comment:
but originally you intended it to be the other way around, right?
So let me switch all of these ;-)